.github/workflows/ci.yml

```diff
@@ -31,6 +31,7 @@ jobs:
         with:
           toolchain: stable
       - run: rustup component add clippy
+      - run: cargo fmt --all --check
```

Comment:
This is a single crate, so why the `--all` flag?

Reply:
Certainly. I will remove it.

Reply:
Fixed.